litestar/openapi/plugins.py

```diff
@@ -0,0 +1,584 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Sequence
+
+import msgspec
+import yaml
+
+from litestar.enums import MediaType, OpenAPIMediaType
+from litestar.handlers import get
+from litestar.serialization import encode_json, get_serializer
+
+if TYPE_CHECKING:
+    from litestar.connection import Request
+    from litestar.router import Router
+
+__all__ = (
+    "OpenAPIRenderPlugin",
+    "RapidocRenderPlugin",
+    "RedocRenderPlugin",
+    "StoplightRenderPlugin",
+    "SwaggerRenderPlugin",
+    "YamlRenderPlugin",
```

Comment:
```suggestion
    "OpenAPIRenderPlugin",
    "RapidocRenderPlugin",
    "RedocRenderPlugin",
    "StoplightRenderPlugin",
    "SwaggerRenderPlugin",
    "YamlRenderPlugin",
    "JsonRenderPlugin",
```

Reply:
Intentionally left the json one undocumented because we always add it. Happy to discuss.

Reply:
I just saw it angry in the tests as undeclared. not pressed about it :D